apps/aragon-fundraising/app/src/screens/Presale.js

```diff
@@ -1,9 +1,12 @@
-import React from 'react'
+import React, { useState } from 'react'
 import styled from 'styled-components'
 import { Badge, Box, Countdown, SafeLink, Button, BREAKPOINTS } from '@aragon/ui'
 import CircleGraph from '../components/CircleGraph'
+import RefundSidePanel from '../components/RefundSidePanel'
 
 export default ({ state }) => {
+  const [refundPanel, setRefundPanel] = useState(false)
+
   const DAY_IN_MS = 1000 * 60 * 60 * 24
```

Comment:
Since we rely on `date-fns` on our codebase, I think it's a good idea to expand it's usage where date calaculations are needed.

cf. https://date-fns.org/v2.2.1/docs/addDays

Reply:
This should be for another PR? Where we we actually get the right time from the smart contract?

Reply:
I pushed a commit to fix this, we'll change the calculation when we'll integrate the presale screen with background script